go/api/commands.go

```diff
@@ -371,4 +371,18 @@ type StringCommands interface {
 	//
 	//[valkey.io]: https://valkey.io/commands/getdel/
 	GetDel(key string) (string, error)
+
+	// Del removes the specified keys. A key is ignored if it does not exist.
+	//
+	// Parameters:
+	//  keys - One or more keys to delete.
+	//
+	// Return value:
+	//  Returns the number of keys that were removed.
+	//
+	// For example:
+	// 	result, err := client.Del("key1", "key2", "key3")
+	//
+	// [valkey.io]: https://valkey.io/commands/del/
+	Del(keys ...string) (int64, error)
```

Comment:
Let's try to maintain the uniformity in the function signature. Either we can use `keys []string` here or pass `keys ... string` at other places.

Reply:
We don't use varargs in other clients, please, use `[]`